weaver/sdks/fabric/interoperation-node-sdk/src/SatpAssetManager.ts

```diff
@@ -0,0 +1,1111 @@
+/*
+ * Copyright IBM Corp. All Rights Reserved.
```

Comment:
Copyright should be removed

Reply:
@outSH Let's skip this as we discussed in the last Cacti maintainers' call.